### DISCUSSION TITLE
Alist桌面版在笔记本重置一次BIOS后会要求重新激活

### DISCUSSION THREAD

**Discussion by imsubei** [AlistGo/alist#4804] [Q&A] [answered]
我是在笔记本上装了一个黑苹果，但是最近电池出现问题，于是只插电使用，每次开机要重置BIOS，现在次次开机都让我激活一次，我以为是登录，结果是使用了一次激活，现在直接告诉我激活次数用完，不能使用了，软件更新之前都没有这样的问题，我也没改过序列号这些三码。有老哥遇到过吗，我也给作者发了邮件询问。

**Comment by xhofe** [comment, marked as answer]
以取消所有激活 你可以按照以下内容来固定主机名：
https://blog.csdn.net/weixin_35379749/article/details/112621779

**Reply by imsubei** [reply]
好的，已运行命令，我现在观察下次重启问题会不会复现。

**Reply by imsubei** [reply]
但是昨天在运行命令未重启之前，激活的还是mac自己更改的主机名，希望取消：192.168.2.13 这个主机名的激活

**Comment by ZhengZY-EE** [comment]
您好，我这边也出现的类似的问题，已经发邮件给您了，也麻烦您帮忙取消一下所有激活，非常感谢！

**Reply by xhofe** [reply]
已取消
